src/data/gateway-data-source.ts

```diff
@@ -61,29 +64,22 @@ export class GatewayDataSource implements ContiguousDataSource {
       path,
     });
 
-    const requestOriginAndHopsHeaders: { [key: string]: string } = {};
-    let hops;
-    let origin;
-    if (requestAttributes !== undefined) {
-      hops = requestAttributes.hops + 1;
-      requestOriginAndHopsHeaders[headerNames.hops] = hops.toString();
-
-      if (requestAttributes.origin !== undefined) {
-        origin = requestAttributes.origin;
-        requestOriginAndHopsHeaders[headerNames.origin] = origin;
-      }
-    } else {
-      hops = 1;
-    }
+    const requestAttributesHeaders =
+      generateRequestAttributes(requestAttributes);
 
     const response = await this.trustedGatewayAxios.request({
       method: 'GET',
       headers: {
         'Accept-Encoding': 'identity',
-        ...requestOriginAndHopsHeaders,
+        ...requestAttributesHeaders?.headers,
       },
       url: path,
       responseType: 'stream',
+      params: {
+        'ar-io-hops': requestAttributesHeaders?.attributes.hops,
+        'ar-io-origin': requestAttributesHeaders?.attributes.origin,
+        'ar-io-version': requestAttributesHeaders?.attributes.nodeRelease,
```

Comment:
Sorry, my bad on the description in the ticket. Let's change this to be `release` instead of `version` and make it `ar-io-origin-release`.